modules/sdk/project-templates/project-templates-workspace/src/main/resources/archetype-resources/gradle.properties

```diff
@@ -55,7 +55,9 @@
     # "liferay.workspace.bundle.token.download" property to true. The default
     # value is the URL for Liferay Portal CE 7.0.4 GA5.
     #
-    #liferay.workspace.bundle.url=https://cdn.lfrs.sl/releases.liferay.com/portal/7.0.4-ga5/liferay-ce-portal-tomcat-7.0-ga5-20171018150113838.zip
+    #if (${liferayVersion} == "7.1")#liferay.workspace.bundle.url=https://cdn.lfrs.sl/releases.liferay.com/portal/7.1.0-m1/liferay-ce-portal-tomcat-7.1-m1-20180223151209360.zip
```

Comment:
if we are setting 7.1-m1 because they have specified 7.1 then we should just set the property.  In your else block you still have a `#` so that means the setting wont be effective.